packages/persistence/internal/exporters/records/index.ts

```diff
@@ -3,12 +3,17 @@ import { Exporter, ExportFormats } from '../index';
 import { CSVRecordsExporter } from './csv';
 import { JSONRecordsExporter } from './json';
 
-export function createRecordsExporter(folder: Folder, format: ExportFormats = 'csv', fileName?: string): Exporter {
+export function createRecordsExporter(
+  folder: Folder,
+  format: ExportFormats = 'csv',
+  fileName?: string,
+  recordType?: string
```

Comment:
Since these two are optional, how about using a configuration object instead? This way we can add more options in the future 🙂

Reply:
Agree 👍